### ISSUE TITLE
"GetBackFocus@ : TopHooker,GetBackFocus,Normal" popup

### ISSUE THREAD

**Issue by earthfront** [brookhong/Surfingkeys#115]
My environment: Ubuntu 16.04, Google Chrome	54.0.2840.59 (Official Build) (64-bit)
Other active extensions: uBlock, Pocket, Quick JavascriptSwitcher, The Great Suspender, Url In Title, Empty New Page Tab, 1-click Timer.

Repro randomly, happened 3 times today. On this time, I was at: http://www.zillow.com/homes/for_sale/globalrelevanceex_sort/47.663537,-115.897522,45.550601,-119.413147_rect/7_zm/

[Image captured.](http://i.imgur.com/SdQdwfz.png)

Why do I think it's surfingkeys? Because inspecting the element leads to the root element:
```
<iframe allowtransparency="true" frameborder="0" scrolling="no" class="sk_ui" src="chrome-extension://gfbliohnnapiefjpjlpjnehglfpaknnc/pages/frontend.html" style="height: 100%; pointer-events: all;"></iframe>
```

"sk_ui" -> surfingKeys if I'm not mistaken.

**Comment by earthfront** [comment]
Let me know if I can provide more info. Thanks for the plugin.

**Comment by brookhong** [comment]
Thanks.
Yes, it is. When does the popup show?  Just after the page loaded without any keystroke? Is the tab active when the popup shows? Or the popup only shows just after you switched to the tab?

**Comment by earthfront** [comment]
I've only noticed it when a tab is opened by ctrl+clicking a link, that is, immediately creating and switching to the new tab. 
It shows immediately. In one case, pressing escape made it go away. In another, I had to reload the page to make it go away.

**Comment by brookhong** [comment]
So you used mouse to open the link? And `Esc` should always be able to dismiss it.

**Comment by earthfront** [comment]
I'm pretty sure I used the mouse, I rarely use [a+f]. 

`Esc` didn't work the first time. Not sure why. Had to reload.

**Comment by slifin** [comment]
I get this quite often on https://inbox.google.com

**Comment by akrasuski1** [comment]
I occasionally get this popup as well. This happens randomly, but it usually happens after using `F` (bound as `gf`) to open a few links in new tabs, then using Chrome's own shortcut Ctrl+Tab to switch to those or Ctrl+W to close original tab.

**Comment by brookhong** [comment]
Fixed in 0.6.4.
